packaging/nnstreamer.spec

```diff
@@ -195,6 +195,13 @@ Group:		Multimedia/Framework
 Requires:	capi-nnstreamer-single-new = %{version}-%{release}
 %description -n capi-nnstreamer-single-new-devel
 Developmental kit for Tizen Native new single-shot NNStreamer API.
+
+%package -n capi-nnstreamer-internal
+Summary:	Tizen internal API to construct the pipeline
+Group:		Multimedia/Framework
+Requires:	%{name} = %{version}-%{release}
+%description -n capi-nnstreamer-internal
+Tizen internal API to construct the pipeline without the permissions.
```

Comment:
Having it named capi-* may confuse other Tizen developers (integrators especially...)

Let's name it nnstreamer-tizen-internal-capi-devel
And add ```Requires: capi-nnstreamer-devel = ...,```